leafsmart-front-end/src/helpers/formats.js

```diff
@@ -0,0 +1,7 @@
+const kebabCase = (string) => {
```

Comment:
will this help with searches like "new york" which need to be "new-york" ?

Reply:
Yep! It's a real generic function, soooo figured I oughtta put it where any component that needs it can call it

Reply:
(It's used for the Cityfacts data, to get facts for places like Red Deer and St John's. :))